.github/workflows/build-test.yaml

```diff
@@ -106,8 +105,14 @@ jobs:
         with:
           username: "${{ env.DOCKERHUB_PUBLIC_USER }}"
           password: "${{ env.DOCKERHUB_PUBLIC_ACCESS_TOKEN }}"
-      - name: "Integration tests"
-        run: "go run mage.go test:integration"
+      - name: "Integration tests with coverage"
+        run: "go run mage.go test:integrationCover"
+      - name: "Coverage"
```

Comment:
I don't love that this step is everywhere, but it works for now. In the future we can probably cache the various `coverage.txt` files with different names and then push them all in one go.